cylc/flow/config.py

```diff
@@ -309,17 +309,19 @@ def __init__(
         self.cfg = self.pcfg.get(sparse=False)
         self.mem_log("config.py: after get(sparse=False)")
 
-        # after the call to init_cyclers, we can start getting proper points.
-        init_cyclers(self.cfg)
-        self.cycling_type = get_interval_cls().get_null().TYPE
-        self.cycle_point_dump_format = get_dump_format(self.cycling_type)
-
         # Running in UTC time? (else just use the system clock)
         if self.cfg['cylc']['UTC mode'] is None:
             set_utc_mode(glbl_cfg().get(['cylc', 'UTC mode']))
+            # This must be set before call to init_cyclers(self.cfg):
+            self.cfg['cylc']['UTC mode'] = get_utc_mode()
         else:
             set_utc_mode(self.cfg['cylc']['UTC mode'])
```

Comment:
Actually, it seems to me that instead of manually getting the global config defaults, this ought to be done in `self.pcfg.get(sparse=False)`, i.e. when expanding the config dict, which currently only takes the suite config defaults?